docs/bills-matching.md

```diff
@@ -0,0 +1,139 @@
+# Bills/transactions matching algorithm
+
+The matching algorithm is in charge of finding matchings between
+`io.cozy.bills` and `io.cozy.bank.operations` documents. This document explains
+how the matching algorithm works.
+
+## The Linker
+
+The algorithm is held by the [`Linker`](), and more precisely its `linkBillsToOperations` method. This method takes 3 parameters:
+
+* An array of `io.cozy.bills` objects
+* An array of `io.cozy.bank.operations` objects
+* An options object
+
+The options object can contain the following properties:
+
+* `minAmountDelta`: defaults to `0.001`
+* `maxAmountDelta`: defaults to `0.001`
+* `pastWindow`: defaults to 15 (days)
+* `futureWindow`: defaults to 29 (days)
+
+These options will determine the default date and amount windows in which a
+transaction should be to match with a bill.
+
+## General flow
+
+When called, this method will do the following (each part will be explained in
+details after):
+
+* Filter bills that are declared as third party payer: these bills have no matching transation since it has been paid by somebody else
```

Comment:
```suggestion
* Filter bills that are declared as third party payer: a third party bill does not have a matching transaction since it has been paid by somebody else
```